ehr/app/src/main/java/org/hl7/gravity/refimpl/sdohexchange/dto/request/TaskStatus.java

```diff
@@ -0,0 +1,19 @@
+package org.hl7.gravity.refimpl.sdohexchange.dto.request;
+
+import lombok.Getter;
+import org.hl7.fhir.r4.model.Task;
+
+/**
+ * Possible values for {@link jdk.tools.jimage.JImageTask.Task} status change.
```

Comment:
Wrong link. CBRO and EHR have same update endpoints. We should sync our code together.